aries-site/src/examples/templates/forms/ShippingExample.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import {
+  Box,
+  Button,
+  CheckBox,
+  Form,
+  FormField,
+  Header,
+  Heading,
+  Main,
+  MaskedInput,
+  Select,
+  Text,
+  TextInput,
+} from 'grommet';
+
+import { FormContainer } from '.';
+import { emailMask, emailValidation, phoneMask, states } from './formHelpers';
+
+export const ShippingExample = () => {
+  const [formValues, setFormValues] = React.useState({});
+
+  // eslint-disable-next-line no-unused-vars
+  const onSubmit = ({ value, touched }) => {
+    // Your submission logic here
+  };
+
+  return (
+    <FormContainer width="medium">
+      <Box gap="medium">
+        <Header
+          direction="column"
+          align="start"
+          gap="xxsmall"
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Heading level={3} margin="none">
+            Shipping
+          </Heading>
+          <Text>for your HPE products</Text>
```

Comment:
The gap between this text and the start of the form feels a bit too spacious in comparison to the design

Reply:
I think since your using Heading its the margin thats already built in making that space the gap is the same so maybe give `margin top= 'none`

Reply:
Good calls. Thank you.

Reply:
fixed